README.md

```diff
@@ -0,0 +1,65 @@
+# Zrythm Flatpak
+
+Flatpak build of [Zrythm](https://www.zrythm.org), a highly automated and intuitive digital audio workstation.
+
+## Plugins
+
+This package supports Flatpak builds of LV2, LXVST, and VST3 plugins. Plugins not installed as Flatpak cannot be used.
+
+You can install Plugins using GNOME Software. Or, to view a list of installable plugins, run:  
+`flatpak install org.freedesktop.LinuxAudio.Plugins//21.08 -y`
```

Comment:
an end user will never see this.

Reply:
Yes, but at least this info exists somewhere for now. Once this is more ready this can be put upstream to docs/manuals somewhere.